org-code-codebuilderapp/app/build.gradle

```diff
@@ -0,0 +1,45 @@
+/*
+ * Usage:
+ * gradle build -> builds the project
+ * gradle bootRun -> sets up a local server -> can be accessed at localhost:8080
+ *
+ * To get your changes into the docker container, do the following
+ * gradle build
+ * cp app/build/libs/app-0.0.1-SNAPSHOT.jar dockerContainer
```

Comment:
Do we want to name our jar something a little more descriptive than 'app'?  Btw, what are calling this service?

Reply:
Good callout! We haven't chosen a name for the service. I'm currently using `codebuilder`, but I'd love suggestions for something more descriptive but still short.
(Oh also, I'm taking your suggestion and updating it to not use app 🙂)

Reply:
I think something with the word Java (to indicate that it processes Java, not necessarily that it's written in Java) would be nice?  JavaRunner, JavaBuilder, JavaCompiler, JavaProcessor, ...?

Reply:
JavaCodeBuilder?

Reply:
Ok. I pitched it to the team and we ended up on `JavaBuilder`. Thank you!